vmdb/app/controllers/application_controller/sysprep_answer_file.rb

```diff
@@ -8,7 +8,7 @@ def upload_sysprep_file
       if params.fetch_path(:upload, :file).respond_to?(:read)
         @edit[:new][:sysprep_upload_file] = params[:upload][:file].original_filename
         begin
-          @edit[:new][:sysprep_upload_text] = MiqProvisionWorkflow.validate_sysprep_file(params[:upload][:file])
+          @edit[:new][:sysprep_upload_text] = SysprepFile.new(params[:upload][:file])
```

Comment:
Is there a potential security issue here since we're sending the params directly to that class?  cc @martinpovolny